src/common/Throttle.cc

```diff
@@ -368,3 +368,61 @@ void OrderedThrottle::complete_pending_ops() {
     ++m_complete_tid;
   }
 }
+
+uint32_t DynamicThrottle::calc_wait_time(uint32_t percentage_empty, 
+                  double &wait_time, bool return_start_delay_if_none) {
+
+  bool need_to_delay = false;
+  if (percentage_empty > 100) {
+    return 1;
+  }
+  if (last_percentage_empty < percentage_empty) {
+    next_threshold = (percentage_empty - (percentage_empty % 5));
```

Comment:
@athanatos I am dividing the entire 100% journal space in 20 slots..
Depending on which slot percentage empty falls in , it will generate a delay time accordingly.
The further away from 100% empty the more delay its generating.